index.html

```diff
@@ -214,6 +215,14 @@
 
         <!-- the map itself -->
         <div id="map" style="display: none;">
+
+        </div>
+        
+        <!-- the footer -->
+        <div class="footer">
+            <img class="footer__share-icon" src="icons/share-icon.svg" onClick="shareWebsiteLink()">
```

Comment:
put on every img element inside footer, -webkit-user-select: none; and cursor: pointer;

Reply:
all images?
maybe you want to select only images with `onclick` property 
```css
.footer img[onclick]{
...
}
```